### PR TITLE
Add missing EME reference pages

### PR DESCRIPTION
**Pull request by teoli2003**
This is part of openwebdocs/project#152

I didn't write an example for these pages, as the whole EME API documentation lacks any. These missing pages are minor features in this API, and it would have been a lack of time to write examples for these missing pages, while the major feature itself has none.

### CODE REVIEW

File: files/en-us/web/api/mediaencryptedevent/index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  - : Creates a new instance of a `MediaEncryptedEvent` object.
+
+## Instance properties
+
+_This interface also inherits properties from its parent, {{domxref("Event")}}._
+
+- {{domxref("MediaEncryptedEvent.initDataType")}} {{ReadOnlyInline}}
+  - : Returns a case-sensitive string with the _type_ of the format of the initialization data found.
+- {{domxref("MediaEncryptedEvent.initData")}} {{ReadOnlyInline}}
+  - : Returns an {{jsxref("ArrayBuffer")}} containing the initialization data found. If there is no initialisation data associated with the format, it returns `null`.
```
Comment (dawei-wang) [inline files/en-us/web/api/mediaencryptedevent/index.md:26]:
```suggestion
  - : Returns an {{jsxref("ArrayBuffer")}} containing the initialization data found. If there is no initialization data associated with the format, it returns `null`.
```

File: files/en-us/web/api/mediaencryptedevent/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+title: MediaEncryptedEvent
+slug: Web/API/MediaEncryptedEvent
+page-type: web-api-interface
+browser-compat: api.MediaEncryptedEvent
+---
+
+{{APIRef("EncryptedMediaExtensions")}}
+
+The **`MediaEncryptedvent`** interface of the [Encrypted Media Extensions API](/en-US/docs/Web/API/Encrypted_Media_Extensions_API) containing the information associated with an {{domxref("HTMLMediaElement/encrypted_event", "encrypted")}} event sent to a {{domxref("HTMLMediaElement")}} when some initialization data is encountered in the media.
```
Comment (dawei-wang) [inline files/en-us/web/api/mediaencryptedevent/index.md:10]:
```suggestion
The **`MediaEncryptedvent`** interface of the [Encrypted Media Extensions API](/en-US/docs/Web/API/Encrypted_Media_Extensions_API) contains the information associated with an {{domxref("HTMLMediaElement/encrypted_event", "encrypted")}} event sent to a {{domxref("HTMLMediaElement")}} when some initialization data is encountered in the media.
```

File: files/en-us/web/api/mediaencryptedevent/initdatatype/index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+title: MediaEncryptedEvent.initDataType
+slug: Web/API/MediaEncryptedEvent/initDataType
+page-type: web-api-instance-property
+browser-compat: api.MediaEncryptedEvent.initDataType
+---
+
+{{APIRef("EncryptedMediaExtensions")}}
+
+The read-only **`initDataType`** property of the {{domxref("MediaKeyMessageEvent")}} returns a case-sensitive string
+describing the type of the initialization data associated with this event.
```
Comment (dawei-wang) [inline files/en-us/web/api/mediaencryptedevent/initdatatype/index.md:11]:
```suggestion
The read-only **`initDataType`** property of the {{domxref("MediaKeyMessageEvent")}} returns a case-sensitive string describing the type of the initialization data associated with this event.
```

**Review by dawei-wang — COMMENTED** [review]
A few typos

File: files/en-us/web/api/htmlmediaelement/encrypted_event/index.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+A {{domxref("MediaEncryptedEvent")}}. Inherits from {{domxref("Event")}}.
+
+{{InheritanceDiagram("MediaEncryptedEvent")}}
+
+## Event properties
+
+- {{domxref("MediaEncryptedEvent.initDataType")}} {{ReadOnlyInline}}
+  - : Returns a case-sensitive string with the _type_ of the format of the initialization data found.
+- {{domxref("MediaEncryptedEvent.initData")}} {{ReadOnlyInline}}
+  - : Returns an {{jsxref("ArrayBuffer")}} containing the initialization data found. If there is no initialisation data associated with the format, it returns `null`.
```
Comment (sideshowbarker) [inline files/en-us/web/api/htmlmediaelement/encrypted_event/index.md:35]:
```suggestion
  - : Returns an {{jsxref("ArrayBuffer")}} containing the initialization data found. If there is no initialization data associated with the format, it returns `null`.
```

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**